packages/web-components/src/components/va-process-list/va-process-list-item.tsx

```diff
@@ -45,9 +45,8 @@ export class VaProcessListItem {
     
     return (
       <Host role="listitem" class='usa-process-list__item'>
-        {
-          checkmark || active || pending ? 
-            <span class='sr-only'>{checkmark ? 'Completed:' : active ? 'Current Step:' : pending ? 'Pending:' : null}</span>
```

Comment:
Removed sr-only text as it sounded redundant on the screen reader to have the text read twice.